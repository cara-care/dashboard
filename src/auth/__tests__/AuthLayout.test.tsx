import React from 'react';
import { render } from '@testing-library/react';
import AuthLayout from '../components/AuthLayout';

describe('<AuthLayout />', () => {
  it('should render correctly', () => {
    const { container } = render(<AuthLayout />);
    expect(container).toMatchSnapshot();
  });
});
