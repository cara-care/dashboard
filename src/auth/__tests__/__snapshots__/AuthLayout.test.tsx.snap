// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLayout /> should render correctly 1`] = `
<div>
  <div
    class="makeStyles-wrapper-1"
  >
    <div
      class="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
    />
  </div>
</div>
`;
